.eslintrc

```diff
@@ -1,4 +1,11 @@
 {
+  "parserOptions": {
+    "ecmaVersion": 11,
+    "sourceType": "module",
+    "ecmaFeatures": {
+        "jsx": true
```

Comment:
Any reasons for this? JSX is preferred for react only I believe. We want to keep this module as flexible and un-opinionated as possible.